inc/namespace.php

```diff
@@ -941,12 +987,18 @@ function enhance_search_query( array $query, array $args, string $type = 'post'
 
 		// Set the full phrase match fuzziness to auto, this will auto adjust
 		// the allowed Levenshtein distance depending on the query length.
-		$query['bool']['should'][1]['multi_match']['fuzziness'] = 'AUTO';
+		// - 0-3 chars = 0 edits.
+		// - 4-6 chars = 1 edit.
+		// - 7+ chars = 2 edits.
+		$query['bool']['should'][1]['multi_match']['fuzziness'] = $fuzziness['distance'];
+		$query['bool']['should'][1]['multi_match']['prefix_length'] = $fuzziness['prefix-length'];
+		$query['bool']['should'][1]['multi_match']['max_expansions'] = $fuzziness['max-expansions'];
+		$query['bool']['should'][1]['multi_match']['fuzzy_transpositions'] = $fuzziness['transpositions'];
 	}
 
 	if ( $mode === 'advanced' ) {
 		$query['bool']['should'] = [
-			get_advanced_query( $args['s'], $search_fields, $strict ),
```

Comment:
Was / is this a bug in v4?

Reply:
No, it's ultimately to pass more useful data to the new filter in the `get_advanced_query()` function. I was looking at using the existing `ep_fuzziness_arg` filter in that function (which needs the whole `$args` array) but opted for providing more control than that offers.